### PR TITLE
RHBRMS-2870 - override of version.org.apache.httpcomponents.httpcore

### PR DESCRIPTION
**Pull request by mbiarnes**
(cherry picked from commit 1b3251f5db803460b55a00bab9f606106833ae03)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2426,10 +2427,25 @@
...
+        <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpmime</artifactId>
         <version>${version.org.apache.httpcomponents.httpclient}</version>
       </dependency>
       <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>${version.org.apache.httpcomponents.httpcore}</version>
+      </dependency>
+      <dependency>
```
Comment (psiroky) [inline pom.xml]: Is this needed? I believe we only need to override the property?

Comment (mbiarnes) [reply pom.xml]: It seems if you only override the property but not explicitly the artifact the artifact coming from jboss-ip-bom has the version of ip-bom.
httpcore was not upgraded to the version we did an override in kie-parent

Comment (psiroky) [reply pom.xml]: This was the case for 6.x, but for 7.x the property override is enough. Just verified that locally with effective pom.

**Comment by mbiarnes** [comment]
@psiroky done

**Comment by psiroky** [comment]
Jenkins execute full downstream build

**Comment by psiroky** [comment]
Failure not related (and the downstream build already failed few times for different reasons). The same change is already on master and 6.5.x, so it should be pretty safe.

**Pull request merged**